scripts/generate-unicode-data

```diff
@@ -130,15 +130,19 @@ main() {
     # ucd-generate is used to compile regexes into DFAs.
     requires ucd-generate
 
+    mkdir -p src/unicode/fsm/
```

Comment:
Hmm, did you find that this line was actually needed? There is already a `src/unicode/fsm/mod.rs` file in that directory, so any checkout should already have that directory created.